packages/azure/createUiDefinition.json

```diff
@@ -0,0 +1,170 @@
+{
+   "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+   "handler": "Microsoft.Azure.CreateUIDef",
+   "version": "0.1.2-preview",
+   "parameters": {
+     "basics": [
+       {}
+     ],
+     "steps": [
+       {
+         "name": "clusterSettings",
+         "label": "Cluster settings",
+         "subLabel": {
+           "preValidation": "Configure cluster",
+           "postValidation": "Done"
+         },
+         "bladeTitle": "Cluster settings",
+         "elements": [
+           {
+             "name": "userObjectId",
+             "type": "Microsoft.Common.TextBox",
+             "label": "userObjectId",
```

Comment:
Could you provide a helpful hint for the users to understand better what these fields are used for?

Reply:
oh, I'll remove this completely. I originally thought I needed to add an access policy for the user to access the Key Vaults at creation time (*e.g.*, if the ever wanted to read a secret value).  The user can create an access policy at any time that allows them to access secrets, so this isn't actually needed.

Reply:
Oh sorry, I meant that we can add a "tooltip" for all of these fields to give a human-readable description. There's a separate property for this:
`"toolTip": "The object Id of the AAD identity that will have access to Key Vault"`

Reply:
Ah, OK -- yes, I'll add tooltips for everything.

Related: https://github.com/celo-org/celo-monorepo/pull/3022